youtube_dl/extractor/extractors.py

```diff
@@ -1667,3 +1667,7 @@
 )
 from .zoom import ZoomIE
 from .zype import ZypeIE
+from .pr0gramm import (
+    Pr0grammIE,
+    Pr0grammStaticIE
```

Comment:
Maintainability (if someone adds a new extractor class, only the new line is different):
```suggestion
    Pr0grammStaticIE,
```